src/server/views/api/queueRetry.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Retry all queue failed jobs
+ * @param {*} req 
+ * @param {*} res 
+ */
+async function handler(req, res) {
+  const { queueName, queueHost } = req.params;
+
+  const { Queues } = req.app.locals;
+
+  const queue = await Queues.get(queueName, queueHost);
+  if (!queue) return res.status(404).send({error: 'queue not found'});
+
+  const promises = [];
+  let jobs;
+  if (queue.IS_BEE) {
+    jobs = await queue.getJobs('failed');
+  } else {
+    jobs = await queue.getFailed();
+  }
+  jobs.forEach((job) => promises.push(job.retry()));
```

Comment:
This seems way terser/more readable:
```suggestion
  const promises = jobs.map((job) => job.retry());
```